java/test/jmri/util/JmriJFrameTestBase.java

```diff
@@ -16,15 +15,15 @@
 
     protected JmriJFrame frame = null;
 
+    @DisabledIfSystemProperty(named ="java.awt.headless", matches ="true")
```

Comment:
It would be good to have a space before the strings:
`named = "java.awt.headless", matches = "true"`